pkg/kubelet/dockertools/manager.go

```diff
@@ -48,12 +48,12 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/network/hairpin"
 	proberesults "k8s.io/kubernetes/pkg/kubelet/prober/results"
 	"k8s.io/kubernetes/pkg/kubelet/qos"
-	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
```

Comment:
In my opinion, I am ok with this change since with Shaya's new change, kubetype is denoted for the common types for entire kubernetes while pkg/kubelet/types only includes kubelet internal types.